lib/client/keystore_test.go

```diff
@@ -80,6 +80,24 @@ func TestListKeys(t *testing.T) {
 	require.Equal(t, samKey.Pub, skey.Pub)
 }
 
+func TestEmptyTeleportClusterNameIsNotAnError(t *testing.T) {
+	s, cleanup := newTest(t)
+	defer cleanup()
+
+	// Given a key store with a valid directory structure
+	host := "some-host"
+	user := "zaphod"
+	key := s.makeSignedKey(t, false)
+	require.NoError(t, s.addKey(host, user, key))
+
+	// When I attempt to enumerate the DB keys with an empty teleport
+	// cluster name
+	_, err := s.store.GetKey(host, user, WithDBCerts("", ""))
```

Comment:
I'd add `WithKubeCerts("", "")` as well to make sure both paths are covered by the test.

Reply:
Done